include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -10838,6 +10838,18 @@
       "capabilities" : [ "TernaryBitwiseFunctionINTEL" ],
       "version" : "None"
     },
+    {
+      "opname" : "OpUntypedVariableLengthArrayINTEL",
+      "class"  : "@exclude",
+      "opcode" : 6244,
+      "operands" : [
+        { "kind" : "IdResultType" },
+        { "kind" : "IdResult" },
+        { "kind" : "IdRef", "name" : "Lenght" }
```

Comment:
```suggestion
        { "kind" : "IdRef", "name" : "Element Type" },
        { "kind" : "IdRef", "name" : "Length" }
```

Reply:
`OpVariableLengthArrayINTEL` also has the typo

Reply:
Thanks! Though I'm not sure, if I see a typo in OpVariableLengthArrayINTEL.

Reply:
ah, thanks! my classical mistake, hope I'll learn how to spell it correctly one day